test-plans/Makefile

```
@@ -0,0 +1,10 @@
+all: ping-image.tar
+
+ping-image.tar: PingDockerfile go.mod go.sum cmd/ping/
+	cd .. && docker build -t go-libp2p-head -f test-plans/PingDockerfile .
```

Please double check this suggestion.
```suggestion
	cd .. && docker build -t go-libp2p-head -f PingDockerfile .
```

---

This is run from the parent directory. This change would break it.